app/assets/javascripts/services/event_notifications_service.js

```diff
@@ -91,6 +91,8 @@ angular.module('ManageIQ').service('eventNotifications', ['$timeout', '$window',
           });
 
           updateUnreadCount(events);
+          const totalUnreadCount = state.groups.reduce((sum, group) => (sum + group.unreadCount), 0);
```

Comment:
I don't think that we can use `Array.prototype.reduce` in old JS. This is not ES6 and its not being compiled by webpack.

Reply:
es6-shim. We can use any ES6 libary feature just fine, only the syntax ones have to go through babel.

Reply:
@fhlavac this is the one ^.. `reduce` is fine, but arrow functions are new *syntax*

Reply:
Oh, and also `const`! :)